docker-compose.yml

```diff
@@ -54,6 +79,26 @@ services:
       - ./data/media:/data/media
     depends_on:
       - postgresql
+      - redis
+    networks:
+      default:
+      lms_outside:
+        aliases:
+          - joanie
+
+  celery:
+    user: ${DOCKER_USER:-1000}
+    image: joanie:production
+    command: ["celery", "-A", "joanie.celery_app", "worker", "-l", "INFO"]
+    environment:
+      - DJANGO_CONFIGURATION=ContinuousIntegration
+    env_file:
+      - env.d/development/common
+      - env.d/development/postgresql
+    ports:
+      - "8071:8000"
```

Comment:
Copy pasta/failed ?

Reply:
Exactly, there is no need to define ports for celery